IaC/ps57.cd/init.groovy.d/cloud.groovy

```diff
@@ -180,13 +180,31 @@ initMap['micro-amazon'] = '''
             sudo mount ${DEVICE} /mnt
         fi
     fi
+    SYSREL=$(cat /etc/system-release | tr -dc '0-9.'|awk -F'.' '{print $1}')
     until sudo yum makecache; do
         sleep 1
         echo try again
     done
-    sudo yum -y install java-1.8.0-openjdk git || :
-    sudo yum -y install aws-cli || :
-    sudo yum -y remove java-1.7.0-openjdk || :
+    if [[ $SYSREL -eq 2 ]]; then
+        sudo amazon-linux-extras install epel -y
+    else
+        sudo yum -y install epel-release
+    fi
+    sudo yum -y install java-1.8.0-openjdk git p7zip || :
+    sudo yum -y remove java-1.7.0-openjdk aws-cli || :
+
+    if ! $(aws --version | grep -q 'aws-cli/2'); then
+        find /tmp -maxdepth 1 -name "*aws*" | xargs sudo rm -rf
```

Comment:
why not just sudo rm -rf /tmp/*aws* ?

Reply:
Because we rm -rf that dir only if it exists, usually it doesn't. 

All Jenkins have this command https://github.com/Percona-Lab/jenkins-pipelines/blob/master/IaC/rel.cd/init.groovy.d/cloud.groovy#L135

Reply:
well in case there is symlink or file which will match to *aws* it also will be removed.  And you need to remove aws dir only without * pattern
So why not 
if [ -d /tmp/aws ]; then
  sudo rm -rf /tmp/aws
fi

it will return only needed directory and will not touch other files/directories

Reply:
Ok, I will update it, and on next update of Jenkins, I'll replace that command with block you've suggested